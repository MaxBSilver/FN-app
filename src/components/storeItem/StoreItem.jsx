import React from "react";

export default function StoreItem(props) {
  return (
    <article className="store-item">
      <img src={props.item.images.information} />
    </article>
  );
}
