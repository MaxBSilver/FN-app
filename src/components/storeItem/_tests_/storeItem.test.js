import React from "react";
import { shallow } from "enzyme";
import StoreItem from "../StoreItem";

describe("StoreItem", () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      item: {
        name: "Crimson Axe",
        description: "Wield, smite, repeat.",
        type: "",
        images: { information: "" }
      }
    };
    wrapper = shallow(<StoreItem {...mockProps} />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
