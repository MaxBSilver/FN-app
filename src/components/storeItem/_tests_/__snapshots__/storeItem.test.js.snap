// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CosmeticItemContainer should match snapshot 1`] = `undefined`;

exports[`StoreItem should match snapshot 1`] = `ShallowWrapper {}`;
